### PR TITLE
chore: update contract & test for alpha release: typescript, imports

### PR DESCRIPTION
**Pull request by Chris-Hibbert**

### CODE REVIEW

File: contract/test/test-contract.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+
+// eslint-disable-next-line import/no-extraneous-dependencies
 import test from 'ava';
-// xxeslint-disable-next-line import/no-extraneous-dependencies
+// eslint-disable-next-line import/no-extraneous-dependencies
 import bundleSource from '@agoric/bundle-source';
 
 import { E } from '@agoric/eventual-send';
-import { evalContractBundle } from '@agoric/zoe/src/contractFacet/evalContractCode';
+import harden from '@agoric/harden';
```
Comment (michaelfig) [inline contract/test/test-contract.js:11]: `harden` is a global created by `@agoric/install-ses` and so it should not be imported here.  If it is not declared, then please either add `"globals": { "harden": "readonly" }` to eslint config, or add `/* global harden */` to the top of this file.

File: contract/test/test-contract.js [hunk 2]
```diff
@@ -1,17 +1,17 @@
-// @ts-nocheck
+// @ts-check
 
 import '@agoric/install-ses';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (michaelfig) [inline contract/test/test-contract.js:5]: Please turn off `"import/no-extraneous-dependencies"` in `package.json`.

**Review by michaelfig — APPROVED** [review]
LGTM, after minor eslint cleanups.

File: contract/test/test-contract.js (hunk 1 above)
Comment (Chris-Hibbert) [reply contract/test/test-contract.js]: done

File: contract/test/test-contract.js (hunk 2 above)
Comment (Chris-Hibbert) [reply contract/test/test-contract.js]: done

**Pull request merged**